inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/CycleUnitType.java

```diff
@@ -21,5 +21,6 @@ public class CycleUnitType {
 
     public static final String DAY = "D";
     public static final String HOUR = "h";
+    public static final String MINUTE = "m";
```

Comment:
DAY and REAL_TIME are in upper case, while HOUR and MINUTE are in lower case. 
Plz keep consistency.